test/backend_tests.cc

```diff
@@ -416,8 +416,8 @@ TEST_CASE("backends report execution time iff profiling is enabled", "[backend]"
 
 	SECTION("on host tasks") {
 		event = backend->enqueue_host_task(
-		    /* lane */ 0, [&](const box<3>&, const communicator*) { std::this_thread::sleep_for(std::chrono::milliseconds(1)); }, {}, box_cast<3>(box<0>()),
-		    nullptr);
+		    /* lane */
+		    0, [&](const box<3>&, const communicator*) { std::this_thread::sleep_for(std::chrono::milliseconds(1)); }, {}, box_cast<3>(box<0>()), nullptr);
```

Comment:
This is very silly. I couldn't find a way to configure this behaviour with a quick search.

Reply:
One idea: would putting the comment *behind* the 0 make it behave less silly? If so, we could standardize on doing that.